source/_components/mqtt.markdown

```diff
@@ -275,6 +275,13 @@ switch:
   platform: mqtt
   state_format: 'json:somekey[0].value'
 ```
+It is also possible to extract JSON values by using a value template:
+
+```yaml
+switch:
+  platform: mqtt
+  value_template: '{{ value_json.somekey[0].value }}'
```

Comment:
Templates must be wrapped with `{% raw %}...{% endraw %}` otherwise they will be rendered and empty.